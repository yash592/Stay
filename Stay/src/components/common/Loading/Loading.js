import React, { Component } from "react";
import { View, Text, Image } from "react-native";

const Loading = () => (
  <View style={{ flex: 1, justifyContent: "center", alignItems: "center" }}>
    <Image
      source={{ uri: "https://i.imgur.com/u6kVKHP.gif" }}
      style={{ width: 100, height: 100 }}
    />
  </View>
);

export { Loading };
