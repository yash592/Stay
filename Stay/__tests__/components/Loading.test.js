import "react-native";
import renderer from "react-test-renderer";
import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { configure } from "enzyme";

configure({ adapter: new Adapter() });
import { Loading } from "../../src/components/common/Loading";
import { shallow } from "enzyme";

describe("Loading component renders correctly", () => {
  it("Shallow rendering", () => {
    const component = shallow(<Loading />);
    expect(component).toMatchSnapshot();
  });
});
