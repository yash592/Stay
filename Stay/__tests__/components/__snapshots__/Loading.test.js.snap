// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders correctly Shallow rendering 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://i.imgur.com/u6kVKHP.gif",
      }
    }
    style={
      Object {
        "height": 100,
        "width": 100,
      }
    }
  />
</View>
`;
